File: week_2__arrays/scores.c
#include <cs50.h>
#include <stdio.h>

const int N = 5;

int *get_user_input(void);
float find_average(int array[]);

int main(void) {
  int *scores = get_user_input();
  float average = find_average(scores);
  printf("You average is: %f\n", average);
  return 0;
}

int *get_user_input(void) {
  int scores[N];
  for (int i = 0; i < N; i++) {
    scores[i] = get_int("Please enter grade: ");
  }
  return scores;
}

float find_average(int array[]) {
  int sum = 0;
  for (int i = 0; i < N; i++) {
    sum += array[i];
  }
  return sum / (float)N;
}

File: week_6__python/scores.py
def main():

  second_grade = int(input("Second grade, please: "))
  third_grade = int(input("Third grade, please: "))

  scores = [72, second_grade]
  scores.append(third_grade)
  print(f"Your first grade is {scores[0]}")

  average_score = sum(scores)/len(scores)

  print(f"{scores[0]}, {scores[1]}, {scores[2]} average {average_score}")

main()